src/main/java/duke/Task.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+
+public class Task {
+    protected String description;
```

Comment:
This is a well defined and simple to read class. Again, you could ensure space before the `{`.